lib/minesweeper_2pl/boardcli.rb

```diff
@@ -0,0 +1,150 @@
+module Minesweeper
+  class BoardCli
+
+    attr_accessor :message, :show_bombs
+
+    NEWLINE = "\n"
```

Comment:
Great use of constants for naming these cryptic strings! What would have to change if I wanted to pass in my own configuration to display the board differently?